src/main/config/azure-blob-application.properties

```diff
@@ -30,10 +30,14 @@ yarn.package.path=file://${basedir}/target/${project.artifactId}-${pom.version}-
 # StreamApplication class
 app.class=samza.examples.azure.AzureBlobApplication
 
-#Azure blob essential configs
+# Azure blob essential configs
 systems.azure-blob-container.samza.factory=org.apache.samza.system.azureblob.AzureBlobSystemFactory
 sensitive.systems.azure-blob-container.azureblob.account.name=your-azure-storage-account-name
 sensitive.systems.azure-blob-container.azureblob.account.key=your-azure-storage-account-key
 
+# Config Loader
+job.config.loader.factory=org.apache.samza.config.loaders.PropertiesConfigLoaderFactory
+job.config.loader.properties.path=./__package/config/azure-blob-application.properties
```

Comment:
If I understand correctly, in the past, it was not necessary to have this config file in the application package on the YARN containers. Was the `samza-hello-samza` build already set up to include the properties file into the application package?
This may also be something you would need to call out in the migration documentation.

Reply:
Yes, `samza-hello-samza` build tarball already include config directory.

Agree, the migration guide for 1.5 need detailed migration instructions on this.